content/docs/start/visualization.md

```diff
@@ -0,0 +1,113 @@
+# Visualization with Plots
+
+In this section, we'll add visualization to the [`example-dvc-experiments`][ede]
+project. If you would like to try these yourself, please refer to the project
+[README] about how to install.
+
+[ede]: https://github.com/iterative/example-dvc-experiments
+[readme]:
+  https://github.com/iterative/example-dvc-experiments/blob/main/README.md
+
+### Creating plots from text with DVC
+
+A useful plot to show the classification performance is the confusion matrix. In
+order to produce it, DVC expects a CSV **plot file** in the form:
```

Comment:
As of now we always use plural in this term. Not sure if ti's best but we can discuss separately if needed. For now let's be consistent?

```suggestion
order to produce it, DVC expects a CSV **plots file** in the form:
```